usix/spacewalk-usix.changes

```diff
@@ -1,3 +1,5 @@
+- Fix StringType as str on Python 3
```

Comment:
This line should be more descriptive for customers, but I am not really sure what we can say.